src/plugins/console/common/constants/api.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const API_BASE_PATH = '../api/console/proxy';
```

Comment:
Base path should only be the base path without any extra routes on it, so that later on in our code we can refer these as `${API_BASE_PATH}/proxy` for example. This is a common pattern that we use in pretty much all of the other apps.

So I think we would want this to be:

```javascript
export const API_BASE_PATH = '/api/console';
```